### DISCUSSION TITLE
cant find a donation link, do u have one?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#130] [General]
cant find a donation link, do u have one?

**Comment by euzu** [comment]
No i dont have a donation link.

**Reply by aniel300** [reply]
u planning on getting one?

**Reply by euzu** [reply]
no, never taught about.

**Reply by aniel300** [reply]
ok if u ever do let me know so i can show u some appreciation and consideration

**Reply by iceman73** [reply]
Same here I was looking yesterday as well

**Comment by euzu** [comment]
ok, thanks.

**Reply by aniel300** [reply]
any update on this?

**Reply by aniel300** [reply]
ok, closing for now, i don't want to push this nor make it awkward
